awx/ui/client/src/shared/smart-search/smart-search.controller.js

```diff
@@ -111,10 +112,13 @@ function SmartSearchController (
     configService.getConfig()
         .then(config => {
             let version;
-
-            try {
-                [version] = config.version.split('-');
-            } catch (err) {
+            if ($rootScope.BRAND_NAME === 'Tower') {
```

Comment:
I think checking the inverse might be better here:

if brand_name == awx -> latest, otherwise: attempt to parse out the config version. This helps us avoid a hard-coded `tower` reference in the `awx` project.

Reply:
@jakemcdermott I originally wrote it that way and @mabashian advised me to write it this way.